interfaces/trusted-types.tentative.idl

```diff
@@ -29,7 +29,7 @@ typedef [StringContext=TrustedScriptURL] USVString ScriptURLString;
   Exposed=(Window, Worker),
   SecureContext
 ] interface TrustedTypePolicyFactory {
-    TrustedTypePolicy createPolicy(DOMString policyName, TrustedTypePolicyOptions policyOptions);
+    TrustedTypePolicy createPolicy(DOMString policyName, optional TrustedTypePolicyOptions policyOptions);
```

Comment:
```suggestion
    TrustedTypePolicy createPolicy(DOMString policyName, optional TrustedTypePolicyOptions policyOptions = {});
```

(without the `= {}` is invalid Web IDL these days; I imagine a future webidl2.js roll might start erroring there.)

Reply:
Done, thanks.